source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -990,6 +990,10 @@ namespace AdaptiveCards { namespace XamlCardRenderer
 
         ComPtr<ITextBlock> xamlTextBlock = XamlHelpers::CreateXamlClass<ITextBlock>(HStringReference(RuntimeClass_Windows_UI_Xaml_Controls_TextBlock));
 
+        // ITextBlock2 will be used later on
+        ComPtr<ITextBlock2> xamlTextBlock2;
```

Comment:
[nit] I find it easier to follow when a variable is declared closer to the point of first usage. Thoughts?